packages/amplify_authenticator/lib/src/blocs/auth/auth_bloc.dart

```diff
@@ -146,6 +146,26 @@ class StateMachineBloc {
     }
   }
 
+  Stream<AuthState> _isValidSession() async* {
+    try {
+      bool isValidSession = await _authService.isValidSession;
+      if (isValidSession) {
+        yield const Authenticated();
+      } else {
+        yield AuthFlow.signin;
+      }
+    } on AmplifyException catch (e) {
+      /// On all AmplifyExceptions other than SignedOutException or SessionExpired,
```

Comment:
*nit:* It would be nice to note that isValidSession is catching SignedOutException and SessionExpired. I was a little confused what this comment meant since I didn't see those exceptions handled differently.

Reply:
Done.